odoo/sql_db.py

```diff
@@ -494,14 +485,21 @@ def close(self):
     def autocommit(self, on):
         _logger.debug("TestCursor.autocommit(%r) does nothing", on)
 
+    def _get_env(self):
```

Comment:
hmm, we need to copies of the method because of the testcursor? Maybe it's better to keep a helper function with a `cr` parameter, to avoid bugfixes going only to one of the two. Thoughts?
Refactoring the Cursor class hierarchy is an alternative, but seems overkill and riskier.